userspace/libsinsp/user.cpp

```diff
@@ -597,7 +597,10 @@ bool sinsp_usergroup_manager::user_to_sinsp_event(const scap_userinfo *user, sin
 		// This can happen at startup when containers are
 		// being created as a part of the initial process
 		// scan.
-		scapevt->ts = sinsp_utils::get_current_time_ns();
+		// note: the following is only a convention and
```

Comment:
This should be treated like container events for the dumping scenario. See comment above.